org.ekstep.questionunit-1.0/renderer/plugin.js

```diff
@@ -32,16 +42,18 @@ org.ekstep.contentrenderer.questionUnitPlugin = Plugin.extend({
    */
   showQuestion: function (event) {
     this.preQuestionShow(event);
-
     var template = _.template(this._question.template);
     var questionsetInstance = event.target;
+    var qData = JSON.parse(questionsetInstance._currentQuestion.data)
     $(questionsetInstance._constants.qsElement).html(template({
       question: this._question
     }));
 
     this.postQuestionShow(event);
-
     this.renderMath(event);
+    if(qData.question.urdu){
```

Comment:
If it V1 question, will this work? qData  will be undefined for V1 question right?

Reply:
Yes,
For v1 question the urdu property will not exist inside question object of qData
if block execution will not happen when the conditional statement returns `undefined or null or false or 0`
Hence we can ignore adding any special cases for v1 question